rpc/src/client/subscription.rs

```diff
@@ -53,14 +54,19 @@ pub trait SubscriptionClient {
 /// ```
 ///
 /// [`Event`]: ./event/struct.Event.html
+#[pin_project]
```

Comment:
Thanks for this! 👍  I was wondering when this was going to become necessary 😁

Reply:
FWIW, I initially did it the unsafe and manual way, but that unsafe block kept me most of the following night, so I figured I'd rather trust a real Rust dev and take the (admittedly very small) compile-time + code size penalty with `pin-project`.

Reply:
Before/after: https://github.com/informalsystems/tendermint-rs/pull/681/commits/7c8f4448e23ca79ce2a12eba87ac5012a98ac0c4